content/scout/policy/remediation.md

```diff
@@ -0,0 +1,133 @@
+available recommendations.
+
+## Outdated base image remediation
+
+The **Outdated base images** policy checks whether the base image you use is
+up-to-date. The recommended actions displayed in the remediation side panel
+depend on how much information Docker Scout has about your image. The more
+information that's available, the better the recommendations.
+
+The following scenarios outline the different recommendations depending on the
+information available about the image.
+
+### No provenance attestations
+
+For Docker Scout to be able to evaluate this policy, you must add [provenance
+attestations](../../build/attestations/slsa-provenance.md) to your image. If
+your image doesn't have provenance attestations, compliance is undeterminable.
+
+When provenance attestations are unavailable, Docker Scout provides generic,
+best-effort recommendations in the remediation side panel. These
+recommendations estimate your base using information from image analysis
+results. The base image version is unknown, but you can manually select the
+version you use in the remediation side panel. This lets Docker Scout evaluate
+whether the base image detected in the image analysis is up-to-date with the
+version you selected.
+
+### Provenance attestations available
+
+With provenance attestations added, Docker Scout can correctly detect the base
+image version that you're using. The version found in the attestations is
+cross-referenced against the current version of the corresponding tag to
+determine if it's up-to-date.
+
+If there's a policy violation, the recommended actions show how to update your
+base image version to the latest version, while also pinning the base image
+version to a specific digest. For more information, see [Pin base image
+versions](../../develop/develop-images/guidelines.md#pin-base-image-versions).
+
+### GitHub integration enabled
+
+If you're hosting the source code for your image on GitHub, you can enable the
+[GitHub integration](../integrations/source-code-management/github.md). This
+integration enables Docker Scout to provide even more useful remediation
+advice, and lets you initiate remediation for violations directly from the
+Docker Scout Dashboard.
+
+With the GitHub integration enabled, you can use the remediation side panel to
+raise a pull request on the GitHub repository of the image. The pull request
+automatically updates the base image version in your Dockerfile to the
+up-to-date version.
+
+This automated remediation pins your base image to a specific digest, while
+helping you stay up-to-date as new versions become available. Pinning the base
+image to a digest is important for reproducibility, and helps avoid unwanted
+changes from making their way into your supply chain.
+
+For more information about base image pinning, see [Pin base image
+versions](../../develop/develop-images/guidelines.md#pin-base-image-versions).
+
+Enabling the GitHub integration also allows Docker Scout to visualize the
```

Comment:
Same thing in here, right now on the side panel we're just tracking the PR being opened and merged. Because as soon as you push the new image that contains the correct digest, the violation would disappear from the list.